written_communications.php

```diff
@@ -75,9 +75,9 @@
 $getHeadline  = admFuncVariableIsValid($_GET, 'headline', 'string', array('defaultValue' => $gL10n->get('PLG_WC_CREATE_WRITTEN_COMMUNICATIONS')),false);
 $getActiveRole  = admFuncVariableIsValid($_GET, 'active_role', 'bool', array('defaultValue' => true));
 //* Check if own templates are available and set template path
-if(is_dir('../../adm_my_files/download/MSWord_Templates'))
+if(is_dir('../../adm_my_files/download_clb/MSWord_Templates'))
 {
-    $dir = '../../adm_my_files/download/MSWord_Templates';
+    $dir = '../../adm_my_files/download_clb/MSWord_Templates';
```

Comment:
**download_clb** is your specific download folder. Use `TableFolder::getRootFolderName()` to get the organization specific download folder.

e.g.
`'../../adm_my_files/'. TableFolder::getRootFolderName() .'/MSWord_Templates'`